androidtv/constants.py

```diff
@@ -310,88 +316,128 @@
 APP_FRANCE_TV = 'fr.francetv.pluzz'
 APP_GOOGLE_CAST = 'com.google.android.apps.mediashell'
 APP_GOOGLE_TV_LAUNCHER = 'com.google.android.apps.tv.launcherx'
+APP_HAYSTACK_NEWS = 'com.haystack.android'
 APP_HBO_GO = 'eu.hbogo.androidtv.production'
 APP_HBO_GO_2 = 'com.HBO'
 APP_HULU = 'com.hulu.plus'
 APP_IMDB_TV = 'com.amazon.imdb.tv.android.app'
+APP_IPTV = 'ru.iptvremote.android.iptv'
 APP_IPTV_SMARTERS_PRO = 'com.nst.iptvsmarterstvbox'
 APP_JELLYFIN_TV = 'org.jellyfin.androidtv'
 APP_KODI = 'org.xbmc.kodi'
+APP_LIVE_CHANNELS = 'com.google.android.tv'
+APP_MIJN_RADIO = 'org.samsonsen.nederlandse.radio.holland.nl'
 APP_MOLOTOV = 'tv.molotov.app'
+APP_MX_PLAYER = 'com.mxtech.videoplayer.ad'
 APP_NETFLIX = 'com.netflix.ninja'
+APP_NLZIET = 'nl.nlziet'
+APP_NOS = 'nl.nos.app'
+APP_NPO = 'nl.uitzendinggemist'
 APP_OCS = 'com.orange.ocsgo'
 APP_PLAY_GAMES = 'com.google.android.games'
 APP_PLAY_MUSIC = 'com.google.android.music'
 APP_PLAY_STORE = 'com.android.vending'
 APP_PLAY_VIDEOS = 'com.android.videos'
 APP_PLEX = 'com.plexapp.android'
 APP_PRIME_VIDEO = 'com.amazon.amazonvideo.livingroom'
-APP_PRIME_VIDEO_2 = 'com.amazon.firebat'
+APP_PRIME_VIDEO_FIRETV = 'com.amazon.firebat'
 APP_SMART_YOUTUBE_TV = 'com.liskovsoft.videomanager'
 APP_SPORT1 = 'de.sport1.firetv.video'
 APP_SPOTIFY = 'com.spotify.tv.android'
 APP_STEAM_LINK = 'com.valvesoftware.steamlink'
 APP_SYFY = 'com.amazon.webapps.nbc.syfy'
+APP_T2 = 'tv.perception.clients.tv.android'
+APP_TED = 'com.ted.android.tv'
+APP_TUNEIN = 'tunein.player'
 APP_TVHEADEND = 'de.cyberdream.dreamepg.tvh.tv.player'
-APP_TWITCH = 'tv.twitch.android.viewer'
-APP_TWITCH_APP = 'tv.twitch.android.app'
+APP_TWITCH = 'tv.twitch.android.app'
+APP_TWITCH_FIRETV = 'tv.twitch.android.viewer'
```

Comment:
It seems that **tv.twitch.android.viewer** is the identifier for an old version (last update seems to be in 2015) of Twitch which apparently still comes installed with Amazon devices.